File: page_body/src/main/java/com/cangwang/service/BodyService.java
package com.cangwang.service;

import android.app.Service;
import android.content.Intent;
import android.os.IBinder;
import android.support.annotation.Nullable;


/**
 * Created by air on 2017/2/19.
 */

public class BodyService extends Service{
    @Nullable
    @Override
    public IBinder onBind(Intent intent) {
        return null;
    }

    @Override
    public void onCreate() {
        super.onCreate();
    }

    @Override
    public void onDestroy() {
        super.onDestroy();
    }


}

File: page_body/src/main/java/com/cangwang/service/BodyService.kt
package com.cangwang.service

import android.app.Service
import android.content.Intent
import android.os.IBinder

/**
 * Created by air on 2017/2/19.
 */

class BodyService : Service() {
    override fun onBind(intent: Intent): IBinder? {
        return null
    }

    override fun onCreate() {
        super.onCreate()
    }

    override fun onDestroy() {
        super.onDestroy()
    }


}
